src/main/java/com/google/api/codegen/nodejs/NodeJSGapicContext.java

```diff
@@ -332,9 +346,15 @@ private String returnTypeComment(Method method, MethodConfig config) {
     for (Field field : config.getRequiredFields()) {
       paramTypesBuilder.append(fieldParamComment(field, null, false));
     }
+    paramTypesBuilder.append(
+        "@param {Object=} options\n"
+            + "  Optional parameters. You can override the default settings for this call, e.g, timeout,\n"
+            + "  retries, paginations, etc. See [gax.CallOptions]{@link "
```

Comment:
This link does not work for me. (It pulls up a page, but there is no `#CallOptions` section.)